### PR TITLE
Several minor adjustments

### PR DESCRIPTION
**Pull request by bastelfreak**
* proper layout in the appendix
* proper shortcuts in the first chapters

### CODE REVIEW

File: thesis-de/content-de.tex [hunk 1]
```diff
@@ -3,30 +3,29 @@ \section{Einführung}
...
-mit großen Datenmengen und dessen Methodiken beschäftigt. Sehr schnell entstand
-aus diesen beiden Gesprächsinhalten die Softwareidee, welche in diesem
-Dokument vorgestellt wird. Da Herrn Meusel und Herrn Luis der Aufwand
-dieser Lösung bewusst war und beide keine Expertise im Bereich Frontend
-Entwicklung, bzw. Datenvisualisierung besaßen, stellten Sie die Idee ihren
-Mitstudenten vor. Herr Reuter zeigte dabei hohes Interesse und war vorallem
-vom englischsprachigen Teil des Projektes überzeugt. Die von den Dozenten
-definierten Vorgaben waren somit erfüllt und das Projekt konnte für das
-fünfte und sechste Semester angemeldet werden.
+Die Zusammensetzung dieses Teams entstand während der rugulären Unterrichtszeit
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: regulären

File: thesis-de/content-de.tex [hunk 2]
```diff
@@ -3,30 +3,29 @@ \section{Einführung}
...
+im dritten Semester. Herr Meusel und Herr Luis sprachen während einem
+Abendessen über die Inhalte ihres täglichen Arbeitstages. Herr Meusel arbeitete
+zu diesem Zeitpunkt an einem Projekt im Bereich Cloud Hosting und Herr Luis war
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: Cloud-Hosting

File: thesis-de/content-de.tex [hunk 3]
```diff
@@ -3,30 +3,29 @@ \section{Einführung}
...
+im Bereich von Datenanalysen mit großen Datenmengen und dessen Methodiken
+beschäftigt. Sehr schnell entstand aus diesen beiden Gesprächsinhalten die
+Softwareidee, welche in diesem Dokument vorgestellt wird. Da Herrn Meusel und
+Herrn Luis der Aufwand dieser Lösung bewusst war und beide keine Expertise im
+Bereich Frontend Entwicklung, bzw. Datenvisualisierung besaßen, stellten Sie
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: Frontendentwicklung

File: thesis-de/content-de.tex [hunk 4]
```diff
@@ -45,46 +44,40 @@ \section{Einführung}
...
-welche eine voll- oder halbautomatische Optimierung von ganzen Rechenzentren
-oder Servergruppen übernehmen kann. Entsprechend versuchen
-Systemadministratoren weiterhin auf ihren zugewiesenen Systemgruppen eine
-optimale Leistung zu erzielen. Dabei fehlt diesen Personen eine Gesamtsicht
-auf Ihre oder andere Servergruppen, entsprechend fällt der Nutzen der
-getätigten Optimierungen im Vergleich sehr gering aus.
-
-Das Problem der fehlenden Übersicht soll in dieser Projektarbeit gelöst
-und dokumentiert werden. Das genaue Ziel ist die Entwicklung von einem
+Kostenoptimierungen. Durch einen neu entstandenen Trend des Cloud Hostings ist
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: Cloud-Hostings ;)

File: thesis-de/content-de.tex [hunk 5]
```diff
@@ -45,46 +44,40 @@ \section{Einführung}
...
+die optimale Auslastung der Hardwaremittel wichtiger denn je. Jedoch besitzt
+der Markt zu dem aktuellen Zeitpunkt noch keine Management Software, welche
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: Management-Software

File: thesis-de/content-de.tex [hunk 6]
```diff
@@ -45,46 +44,40 @@ \section{Einführung}
...
-soll primär von IT-Fachkräften einsehbar sein, sodass die Gesamtperformance
-des Rechenzentrums optimiert werden kann. Je nach Datenschutzeinwilligung
-der Kunden von den einzelnen Computersystem-Anbietern können diese Daten auch
+Rechenzentren für die Ausarbeitung einer Ressourcen-Übersicht. Diese soll
+primär von IT-Fachkräften einsehbar sein, sodass die Gesamtperformance des
+Rechenzentrums optimiert werden kann. Je nach Datenschutzeinwilligung der
+Kunden von den einzelnen Computersystem-Anbietern können diese Daten auch
 sekundär für ein optimiertes Kundengespräch oder Marketing-Kampagnen eingesetzt
 werden. Jedoch handelt es sich dabei nur um ein Nebenprodukt und wird in dieser
 Projektarbeit und Dokumentation ausgeklammert.
```
Comment (bjoernhaeuser) [inline thesis-de/content-de.tex]: s/ausgeklammert/nicht betrachtet/ :D

File: thesis-de/content-de.tex (hunk 1 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 2 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 3 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 4 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 5 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 6 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

**Comment by bastelfreak** [comment]
todo: readd preface + notice about atlassian

**Pull request merged**